content/en/seachat/seachat-manual/02-create-agent/04-agent-information.md

```diff
@@ -0,0 +1,114 @@
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/use-case-examples.png" target="_blank">
+    <img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/use-case-examples.png" alt="image that display the use case options available in SeaChat">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Choose a use case for your AI agent</p>
+</div>
+
+### Description
+A brief description of your AI agent that details your agent's behavior and its reference. Whatever you put inside the description field will show up in agent instructions on the right-hand side of the page. This description will then be used to prompt a language model to generate responses.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/description-preview.png" target="_blank">
+    <img width="100%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/description-preview.png" alt="image showcasing how to write an agent description">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Agent Description</p>
+</div>
+
+### Response Language
+The language the AI agent will use to respond to the user. If the language that the agent will use is not definitive, you can choose **Always match user input language**. 
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/response-language.png" target="_blank">
+    <img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/response-language.png" alt="image showcases the options of response language">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Agent's Response Language</p>
+</div>
+
+### Live Agent
+The live agent feature allows a human agent to intervene in the conversation when needed. If you enable this feature, your customer will have the option to request a live agent during the chat conversation. This requires you to provide a real-human agent that will monitor and respond to the user. 
+
+You can disable this feature by unchecking the box such that your user will not be able to make a request for a live agent during the conversation, while the live agent can still take over the conversation from the **Conversations** dashboard. You can visit our [Live Agent Tutorial](/seachat/seachat-manual/02-create-agent/04-agent-information/) for more detailed explanation of the feature. 
```

Comment:
This part is a bit confusing. 

"... your user cannot request a human agent during the conversation. However, SeaChat admins and agents can intervene in the conversation from the SeaChat Conversation section."